halconfig/settings.js

```diff
@@ -110,6 +110,7 @@ window.spinnakerSettings = {
       },
       useAmiBlockDeviceMappings: false,
     },
+    ecs: ecs,
```

Comment:
Where is the `ecs` variable defined?  This appears to be a lint error.

Reply:
@melbit-michaelw other providers define defaults.  You probably want to do something like this:

```    ecs: {
      defaults: {
        account: 'test',
        region: 'us-east-1',
        iamRole: 'BaseIAMRole',
      },```

I am not sure where these defaults are used though

Reply:
@sbwsg , @BrunoCarrier Added the var definition for ecs, and changed the commit message to be fix based rather than feature.